zone/client_packet.cpp

```diff
@@ -4556,6 +4562,30 @@ void Client::Handle_OP_ChannelMessage(const EQApplicationPacket *app)
 	return;
 }
 
+void Client::Handle_OP_ChangePetName(const EQApplicationPacket *app) {
+	if (app->size != sizeof(ChangePetName_Struct)) {
+		LogError("Got OP_ChangePetName of incorrect size. Expected [{}], got [{}].", sizeof(ChangePetName_Struct), app->size);
+		return;
+	}
+
+	auto payload = (ChangePetName_Struct*)app->pBuffer;
+
+	if (!IsPetNameChangeAllowed()) {
+		payload->response_code = ChangePetNameResponse::NotEligible;
+		QueuePacket(app);
+		return;
+	}
+
+	if (ChangePetName(payload->new_pet_name)) {
+		payload->response_code = ChangePetNameResponse::Accepted;
+	} else {
+		payload->response_code = ChangePetNameResponse::Denied; // not actually needed but included here for completeness
+	}
+
+	QueuePacket(app);
+	return;
```

Comment:
`return;` shouldn’t be necessary since there’s nothing beyond this point. I know the other handlers do this is likely why it’s coded this way, but I’ve been trying to remove them.